docs/data-deduplication.md

```diff
@@ -164,4 +164,25 @@ So Nydus provides a node level CAS system to reduce data downloaded from the reg
 
 The node level CAS system helps to achieve O4 and O5.
 
-# Node Level CAS System (WIP)
+# Node Level CAS System
+Data deduplication can also be achieved when accessing Nydus images. The key idea is to maintain information about data chunks available on local host by using a database.
+When a chunk is needed but not available in the uncompressed data blob files yet, we will query the database using chunk digest as key. 
+If a record with the same chunk digest already exists, it will be reused.
+We call such a system as CAS (Content Addressable Storage).
+
+## Chunk Deduplication by Using CAS as L2 Cache
```

Comment:
This still seems to be an experimental feature, do we still need to consider the `cas.db` record recycling?